app/__init__.py

```diff
@@ -1,12 +1,12 @@
 VERSION = '0.0.1'
 
 
-CONNECTION_ERROR_MSG = "Service connection error"
-REDIRECT_FAILED_MSG = "Failed to redirect to survey"
-SERVER_ERROR_MSG = "Server error"
+CONNECTION_ERROR_MSG = {'text': 'Service connection error. Please try again later.', "clickable": False}
```

Comment:
Can we use a namedtuple here instead of dicts?

```python
from collections import namedtuple

ErrorMessage = namedtuple('ErrorMessage', ['text', 'clickable'])

CONNECTION_ERROR_MSG = ErrorMessage('Service connection error. Please try again later.', False)
...
```

Would then mean updating the [error handlers](https://github.com/ONSdigital/respondent-home-ui/blob/20b702b9fb5d54dcfdbe7b4feebdd7e2fd6637cd/app/error_handlers.py#L59) (and other uses of the errors) a little more straight forward, e.g.:

```python
flash(request, f"{status} " + SERVER_ERROR_MSG.text)
```

Should work the same in the templates.

Reply:
After a bit of investigating, there doesn't seem to be a straightforward way for the template to render data from a namedtuple using dot notation or key lookup, as namedtuples are getting parsed into a list by the template engine.